compiler/rustc_const_eval/src/interpret/place.rs

```diff
@@ -617,19 +621,23 @@ where
 
     /// Computes a place. You should only use this if you intend to write into this
     /// place; for reading, a more efficient alternative is `eval_place_for_read`.
+    #[instrument(skip(self), level = "debug")]
     pub fn eval_place(
         &mut self,
         place: mir::Place<'tcx>,
     ) -> InterpResult<'tcx, PlaceTy<'tcx, M::PointerTag>> {
+        debug!("projection: {:?}", place.projection);
```

Comment:
Please don't add tons of `debug!`. That makes it extremely hard to debug Miri since one just drowns in useless information.

If you must commit them, please make them the lowest verbosity level, `trace!`.